### ISSUE TITLE
Update downgrades on 2018.2

### ISSUE THREAD

**Issue by TWiStErRob** [hsz/idea-gitignore#572]
### Prerequisites

* [x] Plugin is in the latest version
* [x] Issue was not reported yet
* [x] Stack trace (if provided) contains `mobi.hsz.idea.gitignore` package name

### Steps to Reproduce

1. Install AS 3.2 (2018.1.6)
2. Install .ignore plugin
3. Update to AS 3.3 (2018.2.?)

**Expected behavior:**

Update installs newer version, if at all

**Actual behavior:** 

After update, IDEA wants to install an older version:
![image](https://user-images.githubusercontent.com/2906988/51905949-9c9dfe80-23ba-11e9-8e14-0d72c675450e.png)

**Reproduces how often:** 

only had one installation to upgrade, but I suspect 100%

### Versions

**Plugin:** 
```
3.0.0.181 was used on AS 3.2
```

**IDE:** 
```
Android Studio 3.3
Build #AI-182.5107.16.33.5199772, built on December 25, 2018
JRE: 1.8.0_152-release-1248-b01 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
```

**OS:** 
```
Windows 10 10.0
```

**Comment by emanuel-macoun** [comment]
I'm experiencing similar issue. After I did update to IJ 2018.3, I was forced to downgrade .ignore plugin down to version 2.6.2.

**Comment by hsz** [comment]
IDE in version 2018.2+ will be supported again with the upcoming release v3.1.1

**Comment by hsz** [comment]
Fixed and released with v3.1.1

**Comment by NoahAndrews** [comment]
I just got a prompt to downgrade from 3.1.1.191 to 2.6.2 again.
